src/smt/smt_engine.cpp

```diff
@@ -1344,9 +1344,29 @@ void SmtEngine::declareSynthFun(const std::string& id,
     setUserAttribute("sygus-synth-grammar", func, attr_value, "");
   }
   Trace("smt") << "SmtEngine::declareSynthFun: " << func << "\n";
-  // dumping SynthFunCommand from smt-engine is currently broken (please take at
-  // CVC4/cvc4-projects#211)
-  // Dump("raw-benchmark") << SynthFunCommand(id, func, sygusType, isInv, vars);
+
```

Comment:
Can you add a guard around this block of code? 
```
if (Dump.isOn("raw-benchmark"))
{
  ...
}
```
Also maybe add a comment regarding why this is a special case, i.e. "!!! TEMPORARY: We cannot construct a SynthFunCommand since we cannot construct a Term-level Grammar from a Node-level sygus TypeNode. Thus we must print the command using the Node-level utility method for now."

Reply:
Done!